setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = stsci.imagemanip
-version = 1.1.3
+version = 1.1.4
 author = Christopher Hanley
```

Comment:
Haha @chanley is still the main developer, apparently.

Reply:
For-ev-er... 😁